testing/tests/DevExpress.ui.widgets.editors/dropDownEditor.tests.js

```diff
@@ -1049,7 +1049,7 @@ QUnit.test("should have no errors after value change if text editor buttons were
         value: 1,
         fieldTemplate(value) {
             const $textBox = $("<div>").dxTextBox();
-            return $("<div>").text(value + this.option("value")).append($textBox);
+            return $("<div>").text(value + value).append($textBox);
```

Comment:
This change completely change idea of this test. What do you check after this change??

Reply:
Ok, It seems I got it. There is no differece for this test what text will be in the template. So I suggest remove value + value and just luse simple value

Reply:
I have reverted all similar tests. Without any BC